src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.component.ts

```diff
@@ -10,16 +10,18 @@ import { Observable } from 'rxjs/Observable';
     templateUrl: './dot-edit-page-workflows-actions.component.html',
     styleUrls: ['./dot-edit-page-workflows-actions.component.scss']
 })
-export class DotEditPageWorkflowsActionsComponent implements OnInit {
-    @Input() inode: string;
+export class DotEditPageWorkflowsActionsComponent implements OnChanges {
+    @Input() pageState: any;
     @Input() label: string;
 
+    inode: string;
     actionsAvailable: boolean;
     workflowsMenuActions: Observable<MenuItem[]>;
 
     constructor(private workflowsService: DotWorkflowService, private httpErrorManagerService: DotHttpErrorManagerService) {}
 
-    ngOnInit() {
+    ngOnChanges() {
+        this.inode = this.pageState.workingInode;
```

Comment:
ngOnChanges is called every time any input changed, so we have to be sure that the the input changed was 'pageSate'. 

here you can see how:

https://github.com/dotCMS/core-web/blob/master/src/app/view/components/listing-data-table/action-header/action-header.ts#L23